workspace/workflows/jobs.py

```diff
@@ -350,24 +395,44 @@ def get_text_blocks_for_key(args) -> str:
     return json.dumps(blocks)
 
 
+def get_usage_text(args) -> str:
+    return "\n".join(
+        [
+            "Usage for `show [target]`. The behaviour for `show-failed [target]` is the same, but skips repos whose workflows are all successful.",
+            "`show [all]`: Summarise all repos, sectioned by team.",
+            "`show [org]`: Summarise all repos for a known organisation, which is limited to the following shorthands and their full names: `os (opensafely)`, `osc (opensafely-core)` or `ebm (ebmdatalab)`.",
+            "`show [repo]`: Report status for all workflows in a known repo (e.g. `show airlock`) or a repo in a known org (e.g. `show os/some-study-repo`).",
+            "To pass multiple targets, separate them by spaces (e.g. `show os osc` or `show airlock ehrql`).",
```

Comment:
Might be useful to note here that the multiple targets must of the same type - multiple orgs or multiple repos, not a combination